hack/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/devtools/rust-toolset-rhel7:1.43.1 as builder
+
+WORKDIR /opt/app-root/src/
+COPY . .
+RUN bash -c "source /opt/app-root/etc/scl_enable && cargo build --release"
```

Comment:
I don't understand when this switches to the `hack/` directory

Reply:
This runs within `hack/` directory:
```
- context_dir: hack
  dockerfile_path: Dockerfile
  from: rust-toolset
  to: publish
```
in `release` repo